### DISCUSSION TITLE
Is Wi-Fi debugging support publicly tracked?

### DISCUSSION THREAD

**Discussion by beedell-roke** [DP-Hridayan/aShellYou#183] [Q&A] [answered]
I'd like to be notified when it's implemented, since it's what makes aShell useful to me.

**Comment by KrishnaSSH** [comment]
We have not found any libraries that support this functionality, so we are unable to make any promises at this time.

**Reply by beedell-roke** [reply]
> We have not found any libraries that support this functionality.

@KrishnaSSH, I'm surprised you haven't, considering that [the original aShell](https://gitlab.com/sunilpaulmathew/ashell) supports it. Is your application architected such that it is unable to consume what aShell does?

> so we are unable to make any promises at this time.

It *is* listed in the GUI as "coming soon". Perhaps rephrase it, as "under investigation"?

**Comment by DP-Hridayan** [comment]
I haven't been following the original aShell since the first build, I built
it based on v0.9.0 and then worked on it independently. It only takes the
basic Shizuku shell from the original , rest of the things like root,
features , everything are built from scratch..
I will also like to know if there is any open source Wifi debugging
available.

**Comment by KrishnaSSH** [comment, marked as answer]
The Wifi ADB feature has been implemented
